pandas/core/strings.py

```diff
@@ -51,46 +51,61 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
     """
     Concatenate strings in the Series/Index with given separator.
 
+    This function concatenates elements of Series/Index and elements
+    of lists or list-like objects having same length.
+    The concatenation is done at corresponding indices of
+    iterable specified by `Series` when two or more iterables
+    are present. If `others` is not being passed then
+    all values in the Series are concatenated in a single string.
+    The final concatenation is done with a given `sep`
+    and a string is returned, `sep` is optional.
+
     Parameters
     ----------
     others : list-like, or list of list-likes
-      If None, returns str concatenating strings of the Series
+        If None, returns str concatenating strings of the Series.
     sep : string or None, default None
+        If None, concatenates without any separator.
     na_rep : string or None, default None
         If None, NA in the series are ignored.
 
     Returns
     -------
     concat : Series/Index of objects or str
 
+    See Also
+    --------
+    str_split : Split each string in the Series/Index
```

Comment:
str_split -> split (`str_split` is an implementation detail in this case, `split`is the user-facing interface)